tests/README.md

```diff
@@ -7,6 +7,11 @@ The backend test suite can be run outside the Docker container.
 We need all but the main `backend` service running.  Since multiple `backend` instances are allowed, we will simply start everything up
 
     # ENSURE THE IMAGES ARE CLEAN
+    docker rm "/mysql"
+    docker rm "/frontend"
+    docker rm "/rabbitmq" 
+    docker rm "/redis"
+    docker rm "/backend"
     docker-compose down
```

Comment:
I wonder if this command would help shorten this block of commands: ```docker-compose down --rmi local -v```

Reply:
good wondering.  Since I do not know how to recreate the problem, we must stick to what I know works.

Reply:
Your suggestion does not work.  Trying other variations may take forever with no guarantee we will find something shorter.  

```
C:\Users\kyle\code\treeherder2>docker-compose down --rmi local -v
Removing network treeherder2_default
Removing volume treeherder2_mysql_data

C:\Users\kyle\code\treeherder2>    docker-compose up --build
Creating network "treeherder2_default" with the default driver
Creating volume "treeherder2_mysql_data" with default driver
Building backend
Step 1/15 : FROM python:3.7.2
 ---> 2053ca75899e
Step 2/15 : ENV NEW_RELIC_CONFIG_FILE newrelic.ini
 ---> Using cache
 ---> aa16c195b5b0
Step 3/15 : RUN apt-get update && apt-get install -y --no-install-recommends     default-libmysqlclient-dev     gcc     netcat     && rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 104d64da1ca9
Step 4/15 : ENV GECKODRIVER_VERSION='0.24.0'
 ---> Using cache
 ---> 3643d63382d3
Step 5/15 : RUN curl -sSfL --retry 5 "https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz"     | tar -zxC "/usr/local/bin"     && curl -sSfL --retry 5 'https://download.mozilla.org/?product=firefox-beta-latest&lang=en-US&os=linux64'     | tar -jxC "/usr/local/bin"
 ---> Using cache
 ---> b81bdbf1b34a
Step 6/15 : RUN apt-get update && apt-get install -y --no-install-recommends     libgtk-3-0     libdbus-glib-1-2     && rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> fb2ea11814bb
Step 7/15 : ENV MOZ_HEADLESS=1
 ---> Using cache
 ---> 6fac16c6088d
Step 8/15 : ENV PATH="/usr/local/bin/firefox:${PATH}"
 ---> Using cache
 ---> f4e18ff2ebb1
Step 9/15 : ENV SHELLCHECK_VERSION="0.4.6"
 ---> Using cache
 ---> 01fa9b02ee8a
Step 10/15 : RUN curl -sSfL --retry 5 "https://storage.googleapis.com/shellcheck/shellcheck-v${SHELLCHECK_VERSION}.linux.x86_64.tar.xz"     | tar -Jx --strip-components=1 -C /usr/local/bin
 ---> Using cache
 ---> d1d879b24cd1
Step 11/15 : ADD . /app
 ---> Using cache
 ---> 60a804c71345
Step 12/15 : WORKDIR /app
 ---> Using cache
 ---> 566df236c403
Step 13/15 : RUN pip install --no-cache-dir --disable-pip-version-check --require-hashes -r requirements/common.txt
 ---> Using cache
 ---> c3328f6e55a3
Step 14/15 : RUN pip install --no-cache-dir --disable-pip-version-check --require-hashes -r requirements/dev.txt
 ---> Using cache
 ---> ae55c4a33058
Step 15/15 : RUN pip install --no-cache-dir --disable-pip-version-check -r requirements/docs.txt
 ---> Using cache
 ---> 4d187dbfa63f
Successfully built 4d187dbfa63f
Successfully tagged treeherder-backend:latest
Creating frontend ...
Creating rabbitmq ...
Creating redis    ... error
Creating mysql    ...
Creating frontend ... error
ERROR: for redis  Cannot create container for service redis: Conflict. The container name "/redis" is already in use by container "4b87d4d939093102fed7e0bfa4d2ac897cc4a657657bab95bbfe722bdb6714de". You have to remove (or rename) that container to be able to reuse that name.

Creating rabbitmq ... error
n use by container "cdc3618330397a8fa6b5c869dad07adec7102c0e5cf648162abc6b9ec3b83568". You have to remove (or rename) that container to be able to reuse that name.

ERROR: for rabbitmq  Cannot create container for service rabbitmq: Conflict. The container name "/rabbitmq" is already in use by container "72d2542bb883c44f7a133eebd3453affe4e48cff6242d0a32380940b32875b28". You have to remove (or rename) thCreating mysql    ... error

ERROR: for mysql  Cannot create container for service mysql: Conflict. The container name "/mysql" is already in use by container "7ae42a9a220bb14de7fe02f07b2a8f04f88186c7914f436d5c97252a394bf550". You have to remove (or rename) that container to be able to reuse that name.

ERROR: for redis  Cannot create container for service redis: Conflict. The container name "/redis" is already in use by container "4b87d4d939093102fed7e0bfa4d2ac897cc4a657657bab95bbfe722bdb6714de". You have to remove (or rename) that container to be able to reuse that name.

ERROR: for frontend  Cannot create container for service frontend: Conflict. The container name "/frontend" is already in use by container "cdc3618330397a8fa6b5c869dad07adec7102c0e5cf648162abc6b9ec3b83568". You have to remove (or rename) that container to be able to reuse that name.

ERROR: for rabbitmq  Cannot create container for service rabbitmq: Conflict. The container name "/rabbitmq" is already in use by container "72d2542bb883c44f7a133eebd3453affe4e48cff6242d0a32380940b32875b28". You have to remove (or rename) that container to be able to reuse that name.

ERROR: for mysql  Cannot create container for service mysql: Conflict. The container name "/mysql" is already in use by container "7ae42a9a220bb14de7fe02f07b2a8f04f88186c7914f436d5c97252a394bf550". You have to remove (or rename) that container to be able to reuse that name.
ERROR: Encountered errors while bringing up the project.
```

Reply:
The cause of this problem is two directories using Docker:  Even when one is shutdown, the other generates this complaint.